packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -948,17 +948,17 @@ export interface IndexerVrtxSupplySnapshot {
   timestamp: BigDecimal;
   /** VRTX token price in primary quote. */
   vrtxOraclePrice: BigDecimal;
-  /** Total VRTX tokens distributed as staking incentives. */
+  /** Fraction of total VRTX supply distributed as staking incentives.  */
```

Comment:
So fraction as in - `0.1` for 10%?

I didn't know these were fractions tbh - we should reflect that in the naming - ie. `cumulativeIncentivesFrac/Fraction`

Reply:
Yes these are fractions per [this](https://docs.google.com/spreadsheets/d/1Z9uKaP6jr9Dsa_e47ci9otZ8fdqOQFHf4_L91vRFvWo/edit?ref=blog.vertexprotocol.com&gid=639807660#gid=639807660) sheet.